rfc/rfc010-multiple-operators.md

```diff
@@ -0,0 +1,64 @@
+# RFC010 Multiple operator service access
```

Comment:
I thing this document describes a specific subset where the care provider does not administers its own DID document. I think we should add that case here as well or make it explicit that it is about delegated identity.

Reply:
It's still valid for when an care organization manages its own keys. They will probably not use the functionality. The only thing this spec defines is that data MUST be accessible for DID subjects that refer to each other using the `also-known-as` property.